.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/cpp/abseil-cpp"]
+	path = tools/cpp/abseil-cpp
```

Comment:
Maybe "cpp/deps" as this is the main source directory.
As we are touching cpp build system in existing PR https://github.com/google/libphonenumber/pull/2818/files, we would like to review that first.